mgr/dashboard: snapshot schedule added subvolume and translations

ivoalmeida opened the pull request:
Fixes: https://tracker.ceph.com/issues/64372

<!--
  - Please give your pull request a title like

      [component]: [short description]

  - Please use this format for each git commit message:

      [component]: [short description]

      [A longer multiline description]

      Fixes: [ticket URL on tracker.ceph.com, create one if necessary]
      Signed-off-by: [Your Name] <[your email]>

    For examples, use "git log".
-->

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [x] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

[Screencast from 2024-02-09 17-43-52.webm](https://github.com/ceph/ceph/assets/1071297/6cd815eb-f6fe-4aa2-bba4-162437eae4aa)

nizamial09 reviewed (commented):
In the `Retention Policy` row, instead of `44 Yearly, 2 monthly` phrasing for me it makes more sense if we say it like `Yearly: 44 Monthly: 2`. What do you think?

cloudbehl commented:
jenkins retest this please

ivoalmeida commented:
> In the `Retention Policy` row, instead of `44 Yearly, 2 monthly` phrasing for me it makes more sense if we say it like `Yearly: 44 Monthly: 2`. What do you think?

@nizamial09 thanks for the feedback.

@cloudbehl  what do you think?

cloudbehl commented:
I am fine with both. We want to communicate that we will `Retain 44 snapshots Yearly and 2 monthly`

cloudbehl commented:
jenkins test windows

ivoalmeida commented:
> @ivoalmeida title of this PR says `Snapshot schedule subvolume copies`. is that intended? the commit just says `mgr/dashboard: added subvolume and translations`

changed it 👍🏽

cloudbehl commented:
@ivoalmeida Pull Request Checklist is failing
